cmd/soroban-rpc/internal/config/config_test.go

```diff
@@ -0,0 +1,105 @@
+package config
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stellar/go/network"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func ExampleConfig() {
```

Comment:
In `go test`. it's a godoc example. We don't really use them much, but it documents the expected usage, and checks it executes. I had to reverse-engineer how to use the horizon `ConfigOptions`, so I wanted to avoid that for the next poor soul.

Reply:
oh - I see. thanks for clarifying that.